utils/unittest/fixtures.go

```diff
@@ -1720,3 +1728,45 @@ func DKGBroadcastMessageFixture() *messages.BroadcastDKGMessage {
 		Signature:  SignatureFixture(),
 	}
 }
+
+func PrivateKeyFixture(algo crypto.SigningAlgorithm) crypto.PrivateKey {
+	sk, err := crypto.GeneratePrivateKey(algo, SeedFixture(64))
+	if err != nil {
+		panic(err)
+	}
+	return sk
+}
+
+func NodeMachineAccountInfoFixture() bootstrap.NodeMachineAccountInfo {
+	return bootstrap.NodeMachineAccountInfo{
+		Address:           RandomAddressFixture().String(),
+		EncodedPrivateKey: PrivateKeyFixture(crypto.ECDSAP256).Encode(),
+		HashAlgorithm:     bootstrap.DefaultMachineAccountHashAlgo,
+		SigningAlgorithm:  bootstrap.DefaultMachineAccountSignAlgo,
+		KeyIndex:          bootstrap.DefaultMachineAccountKeyIndex,
+	}
+}
+
+func MachineAccountFixture() (bootstrap.NodeMachineAccountInfo, *sdk.Account) {
+	info := NodeMachineAccountInfoFixture()
+
+	bal, err := cadence.NewUFix64("0.1")
+	if err != nil {
+		panic(err)
```

Comment:
Maybe passing a `t *testing.T` parameter and requiring a no error comes cleaner. It also adds an extra level of clarity to the function signature indicating it is a test fixture.

Reply:
https://github.com/onflow/flow-go/pull/1233/commits/1f02253e840adabaa3aadeef16fc8df9d7c7eeb1